package.json

```diff
@@ -185,7 +185,8 @@
             "fortran": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "sml": "cd $dir && sml $fileName",
             "mojo": "mojo run",
-            "erlang": "escript"
+            "erlang": "escript",
+            "pickle": "pkl eval -f yaml $fileName -o $fileNameWithoutExt.yaml"
```

Comment:
It should be `"pkl"` instead of `"pickle"`, per https://github.com/apple/pkl-vscode/blob/66982961b84c3061c95fc8f4a889facdefccc637/package.json#L33

Reply:
Add `cd $dir &&`, otherwise it will fail to find the file if file is not in root of the folder.

Reply:
Applied changes